invenio_communities/errors.py

```diff
@@ -42,3 +42,17 @@ def __init__(self, limit, file_size):
                 )
             )
         )
+
+
+class OpenRequestsForCommunityError(CommunityError):
+    """The provided logo size exceeds limit."""
+
+    def __init__(self, requests):
+        """Initialise error."""
+        super().__init__(
+            _(
+                "There are {requests} request(s) open for this community. Please, resolve all of them before deleting this community.".format(
```

Comment:
With translations, can we add the singular/plural forms, instead of `There are ... request(s)`?

Reply:
will create an issue for that :)